react/components/UtilityFunctionsPixexEvents.js

```diff
@@ -0,0 +1,21 @@
+export const getCategoryFromObjs = (products) => {
```

Comment:
Not mandatory, but it would be great if this file was written in ts. Do you think you can do that?

Reply:
Changed the file type to ts format :)